guides/index_template.md

```diff
@@ -0,0 +1,186 @@
+# Index Template
+
+Index templates are a convenient way to define settings, mappings, and aliases for one or more indices when they are created. In this guide, you'll learn how to create an index template and apply it to an index.
+
+## Setup
+
+Assuming you have OpenSearch running locally on port 9200, you can create a client instance with the following code:
+
+```rust
+// Create a client to make API calls to OpenSearch running on http://localhost:9200.
+let client = OpenSearch::default();
+// Alternatively, you can create a client to make API calls against OpenSearch running on a specific url::Url.
+let url = Url::parse("https://example.com")?;
```

Comment:
Please 

```suggestion
let url = Url::parse("https://localhost:9200")?; 
```

The `example.com` is a real site, the port is required

Reply:
Yes, sorry, corrected it!

Reply:
Hm .. I still see it as it was before ...